lang/lang-extract.py

```diff
@@ -261,13 +274,25 @@ def main():
     ap.add_argument('file', nargs='+', help='Input files')
     args = ap.parse_args()
 
+    for path in args.file:
+        if not Path(path).exists():
+            # assume its regex, search for files that match pattern
+            for file in sorted(PO_DIR.glob(path)):
+                FILE_LIST.append(file)
+        else:
+            FILE_LIST.append(Path(path))
```

Comment:
Overall, since we now don't need the shell script, I would like to do this in ``update-pot.py`` from a style perspective.
No unix tool accepts/processes a glob itself normally, it's always done by calling program.